router/server.go

```diff
@@ -187,21 +187,22 @@ func (s *Server) isLegalRequest(req *http.Request) bool {
 	return true
 }
 
-func (s *Server) resendBucky(orgReq *http.Request) {
-	buf, _ := ioutil.ReadAll(orgReq.Body)
-	newBody := ioutil.NopCloser(bytes.NewBuffer(buf))
-	orgBody := ioutil.NopCloser(bytes.NewBuffer(buf))
-
-	req, err := http.NewRequest("POST", "http://monitor.imqs.co.za/bucky/v1/send", newBody)
-	orgReq.Body = orgBody
+func (s *Server) resendBucky(w http.ResponseWriter, orgReq *http.Request) {
+	buf, err := ioutil.ReadAll(orgReq.Body)
+	if err != nil {
+		return
```

Comment:
If this happens you will return 200 OK and we won't know what went wrong.
Maybe something like: 
```
http.Error(w, "Error reading request body: " + err.Error(), http.StatusBadRequest)
return
```